assets/html/scss/documenter/layout/_sidebar.scss

```diff
@@ -0,0 +1,246 @@
+    @include tablet {
+      // We make sure that the shadow is gone if we've ended up in non-mobile layout
+      // but the .visible class is still toggled.
+      box-shadow: none;
+    }
+  }
+  // In the non-mobile layout, we show the sidebar as a left panel of the dashboard.
+  @include tablet {
+    position: relative;
+    left: 0;
+  }
+
+  // The top bar of the sidebar bar contains:
+  // 1. Logo (optional)
+  .docs-logo {
+    margin-top: 1rem;
+    padding: 0 1rem;
+    > img {
+      @extend .image;
+      max-height: 6rem;
+      margin: auto;
+    }
+  }
+  // 2. The site name, from the sitename keyword (optional)
+  .docs-package-name {
+    flex-shrink: 0;
+    font-size: $sidebar-title-font-size;
+    font-weight: $weight-bold;
+    text-align: center;
+    white-space: nowrap;
+    overflow: hidden;
+    // The padding works around the problem that when you set overflow: hidden, part
+    // of the container gets hidden.
+    padding: 0.5rem 0;
+    // We don't know how wide the text is (the user can provide a _really_ long string to
+    // sitename). So we wrap the name in a <span> which can be dynamically resized with
+    // JS.
+    .docs-autofit {
+      max-width: $sidebar-title-width;
+    }
+  }
+
+  // The version selector (optional). It is hidden by default, but can be toggled by JS
+  // if there actually are some versions to display.
+  .docs-version-selector {
+    border-top: 1px solid $border;
+    display: none;
+    padding: 0.5rem;
+    &.visible {
+      display: inherit;
+    }
+  }
+
+  // Styling, positioning etc. of the menu items
+  ul.docs-menu {
+    @extend .is-scrollable;
+    flex-grow: 1;
+
+    // Heavy-handed approach to disable selecting when you double click on an expandable menu
+    // item
+    user-select: none;
+
+    font-size: $size-normal;
+    border-top: 1px solid $border;
+    // background-color: $background;
+
+    padding-bottom: 1.5rem;
+
+    // First level items
+    > li {
+      // padding-left: 0.5em;
+      > .tocitem {
+        font-weight: bold;
+      }
+    }
+
+    // Second and lower
+    > li li {
+      font-size: 0.95*$size-normal;
+      margin-left: 1em;
+      border-left: 1px solid $border;
+
+      &.is-active {
+        // margin-left: 0;
+      }
+    }
+
+    /* Managing collapsible submenus */
+    input.collapse-toggle {
+        display: none;
+    }
+
+    ul.collapsed {
+        display: none;
+    }
+
+    input:checked ~ ul.collapsed {
+        display: block;
+    }
+
+    label.tocitem {
+      display: flex;
+      .docs-label {
+        flex-grow: 2;
+      }
+      .docs-chevron {
+        display: inline-block;
+        font-style: normal;
+        font-variant: normal;
+        text-rendering: auto;
+        line-height: 1;
+
+        font-size: 0.75*$size-normal;
+        margin-left: 1rem;
+        margin-top: auto;
+        margin-bottom: auto;
+        &::before {
+          @include font-awesome;
+          content: "\f054";
```

Comment:
If you make it specific to levels > 1, then you can put something like
```
margin-left: -1em;
margin-right: 0;
background: $sidebar-background;
```
here to get what I showed on Slack.